.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
 script:
   - commitlint-travis
   - yarn check-format
-  - yarn test
+  - yarn test --ci
```

Comment:
This should be automatically detected, as Jest uses [`is-ci`](https://www.npmjs.com/package/is-ci).

Reply:
ohh, they should add note about this in docs xd

Reply:
> When this option is provided, Jest will assume it is running in a CI environment. This changes the behavior when a new snapshot is encountered. Instead of the regular behavior of storing a new snapshot automatically, it will fail the test and require Jest to be run with --updateSnapshot.

Reply:
but you are right 👍 
https://github.com/facebook/jest/blob/master/packages/jest-util/src/isInteractive.js